specification.md

```diff
@@ -5,71 +5,72 @@ permalink: /specification.html
 title: Specification
 ---
 
-The current version is *2019-09*!
+The current version is *2020-12*!
+The previous version was 2019-09.
 
 Specification documents
 -----------------------
 
-See also the [release notes / change log](draft/2019-09/release-notes.html).
+See also the release notes / change log (Work in progress).
```

Comment:
Lets link to the page, it alread ysays its a work in progress when they get there. One less thing to remember to update.

Reply:
Good plan. I can't make the change right now, but feel free to make a change suggestion =]

Reply:
Eh skip it and merge, this is good.